src/pages/add-restaurant-page/add-restaurant-page.html

```diff
@@ -1,12 +1,6 @@
-<!--
-  Generated template for the AddRestaurantPage page.
-
-  See http://ionicframework.com/docs/v2/components/#navigation for more info on
-  Ionic pages and navigation.
--->
```

Comment:
good to remove this, on the live project we can live here the creation date and the creator, in this case your name XD